### PR TITLE
Hw1 branch

### PR DESCRIPTION
**Pull request by juliapochynok**
Adding Homework 1 (main page)

### CODE REVIEW

**Comment by pavlo-berezin** [comment]
This PR contains 40582 changes. Please fix this.
Hint: add .gitignore :)

**Comment by juliapochynok** [comment]
> This PR contains 40582 changes. Please fix this.
> Hint: add .gitignore :)

.gitignore added

File: src/components/Carousel.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+}
+
+.ItemCar {
+    width: 65vw;
+    height: 60vh;
+}
+
+.d-block {
+    width:100% !important;
```
Comment (pavlo-berezin) [inline src/components/Carousel.css:20]: Consider not using `!important`. It's bad practice.

File: src/components/Carousel.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+.SlideTitle {
+    opacity: 1;
+    white-space: pre;
+    font: oblique;
+    font-family: 'Wire One', sans-serif;
+    text-decoration: none;
+}
+
+.Slide2 {
```
Comment (pavlo-berezin) [inline src/components/Carousel.css:54]: Don't hardcode styles for all slides. Consider unifying them.

File: src/components/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+export default class MainCarousel extends Component {
+    render() {
+        return (
+        
+        <Carousel className="Carousel1" >
+          <Carousel.Item className="ItemCar">
+            <img
+              className="d-block"
+              src="assets/exo_main.jpg"
+              alt="Third slide"
```
Comment (pavlo-berezin) [inline src/components/Carousel.jsx:15]: This is first slide. Isn't it? :)

File: src/components/Footer.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
+
+.Footer1 {
+    position: relative;
+    bottom: 0;
+    height: 10vh;
+    width: 99vw;
```
Comment (pavlo-berezin) [inline src/components/Footer.css:6]: Why not 100?

File: src/components/Footer.jsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    render() {
+        return (
+         <div className="Footer1">
+            <a href="https://www.instagram.com/weareone.exo/?hl=uk"  className="SNS"><img className="SNSPic"  src="assets/insta.webp" alt="inst-logo" /></a>
+            <a href="https://twitter.com/weareoneexo" className="SNS"><img className="SNSPic" src="assets/twt.png" alt="twt-logo" /></a>
+            <a href="https://www.youtube.com/channel/UCzCedBCSSltI1TFd3bKyN6g" className="SNS"><img  className="SNSPic" src="assets/youtube.png" alt="yt-logo" /></a>
+            <a href="http://exo.smtown.com/" className="SNS"><img className="SNSPic" src="assets/logo.png" alt="exo-logo" /></a>
+            <a href="https://www.smtown.com/" className="SNS"><img className="SNSPic" src="assets/SM.png" alt="sm-logo" /></a>
+         </div>
+
```
Comment (pavlo-berezin) [inline src/components/Footer.jsx:15]: Please remove this empty lines.

File: src/components/Home.css [hunk 6]
```diff
@@ -0,0 +1,5 @@
+
+  .person-wrapper {
+    margin-bottom: 7vh;;
```
Comment (pavlo-berezin) [inline src/components/Home.css:3]: one semicolon will be enough :)

File: src/components/Navigator.jsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+export default class Navigator extends Component {
+
+    render() {
+    
+        return (
+        <div className="Header">
+            <nav>
+                <a className="MainContainerA" href="/tests">Tests</a>
+                <a className="MainContainerA"  href="/members">Members</a>
+                <a href="/" className="MainTitle"> <strong>MEET &nbsp;
```
Comment (pavlo-berezin) [inline src/components/Navigator.jsx:14]: Please reformat this part of code.

**Comment by pavlo-berezin** [comment]
image
Please fix those.

**Comment by pavlo-berezin** [comment]
Please improve folder structure. Too much files inside one folder.

**Pull request merged**